lib/logging/json_formatter.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'json'
+
+class JSONFormatter
+  def parse_message(message)
+    begin
```

Comment:
`begin` is unnecessary here.

Reply:
I did not know this. I was able to squash this down quite a bit!

Reply:
@whereismyjetpack yeah, it's one of the "ruby-isms"